src/components/molecules/Progress/Progress.tsx

```diff
@@ -41,12 +43,14 @@ const SProgression = styled.div<IProgressionStyleProps>`
   }
 `;
 
-const Progress: React.FC<IProgressProps> = ({ totalSteps, currentStep, ...rest }) => (
+const Progress: React.FC<IProgressProps> = ({ totalSteps, currentStep, withStep = true, progressColor, ...rest }) => (
   <SProgressBar {...rest}>
-    <SProgression width={`${(100 / totalSteps) * currentStep}%`}>
-      <Text size="small">
-        Step {currentStep} of {totalSteps}
-      </Text>
+    <SProgression width={`${(100 / totalSteps) * currentStep}%`} progressColor={progressColor}>
+      {withStep && (
```

Comment:
Is `withStep` really necessary? You can easily do it outside of the component. If we add this, then the next thing you know you'll have `withPercentage`, `withSlashStepFormat`, etc.

Reply:
The text below the bar is optional, new designs popping up dont have the text below it, i'd agree we could add up to it. I can in a separate PR if you are keen (and also do conditional types validation on props provided) but its quite a bit more work. Let me know.